src/automations/checkInactivity.js

```diff
@@ -137,10 +146,12 @@ async function scrapeInactiveIssues(client, references, issues) {
       owner: repoOwner, repo: repoName, number: issueNumber, per_page: 100
     });
     const com = issueComments.data.slice(-1).pop();
+    const Body = com.body;
+    const user = com.user.login;
 
     // Use end of line comments to check if comment is from template
-    const warning = com ? com.body.endsWith("<!-- inactiveWarning -->") : null;
-    const fromClient = com ? com.user.login === client.cfg.auth.username : null;
+    const warning = com ? Body.endsWith("<!-- inactiveWarning -->") : null;
+    const fromClient = com ? user === client.cfg.auth.username : null;
```

Comment:
Why was this changed?

Reply:
The above two variables were introduced so as to adhere to the less than 80 characters test case since I had earlier nested everything inside the if block, and the indentation introduced cause an overflow from the 80 character limit. Since that's been rectified, i'll rectify these changes now :)